agent/tool-scripts/postprocess/watches-cluster-health-postprocess

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/perl
+
+# Author: Lukas Vlcek (lvlcek@redhat.com)
+#
+# usage: <tool>-postprocess <dir>  dir = directory where tool txt file can be found
+#
+# The purpose of this script is to
+# -output the chart data in JSON format (watches.js)
```

Comment:
It seems that the purpose of a post-processing tool is to convert the generated output of the tool into the appropriate machine and human-readable forms.  However, it seems that the `GenData` model actually emits transformed out, and this module just reads it in.  Perhaps we can say just that?